### ISSUE TITLE
button color bug 

### ISSUE THREAD

**Issue by Yumega** [kalkih/mini-media-player#764]
The buton color is white while the background is also white

type: custom:mini-media-player
entity: media_player.acast
toggle_power: false
volume_stateless: true
volume_step: '2'
group: true
artwork: material
![图像](https://github.com/kalkih/mini-media-player/assets/61570667/291c6727-cf87-407f-b9e0-b49213ba1ed8)

**Comment by spacegaier** [comment]
So you verified that the buttons and icons are actually there, just not clearly visible?

Is there an issue with retrieving the cover artwork or why is the card background white in your case?

**Comment by Yumega** [comment]
Yes, the buttons are there, only not clear, because the button color is white and the background is also white. dont have other problems.

**Comment by spacegaier** [comment]
But why is your background white. From my understanding `artwork: material` should show the cover and fill the rest of the card with a derived color.

But rega5rdless of why, it would be nice if this card correctly reacts tot he actual shown color and reverts to dark font colors for the icons and buttons.

**Comment by Yumega** [comment]
I don't know ,I creat a new one, didn't play anything, the buttons are also not clear, the buttons should be black color, but they are white color here

**Comment by spacegaier** [comment]
Are you using a custom HA theme? I am not able to reproduce the issue on my end.

**Comment by Jaykurb** [comment]
I have this same issue
